.clang-tidy

```diff
@@ -1,11 +1,10 @@
 ---
 Checks: >
   *,
-  -fuchsia-*,
+  -fuchsia-*
```

Comment:
Missing commas really mess things up. You might even want to employ a yml linter for your file. Clang tidy is bad about telling you about errors here

Reply:
Good catch John.

My mistake there. I had temporarily removed `fuchsis` suppressions from the `.clang-tidy` file (which I was not using anyway since i run `clang-tidy` on the command line).

Basically, the `.clang-tidy` should remain unchanged. i will correct this prior to merging the PR.